### PR TITLE
Offensive language action

### PR DESCRIPTION
**Pull request by rnetser**

### CODE REVIEW

**Comment by myakove** [comment]
you should rebase on @AdiZav PRs

**Comment by myakove** [comment]
/verified

File: test_file.py:1:5
    Found offensive words [master]
File: test_file.py:2:9
    Found offensive words [master]
File: test_file.py:6:5
    Found offensive words [slave]

**Comment by myakove** [comment]
Failed CI should be fixed after this PR merged

File: .github/workflows/offensive_language_actions.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Block Offensive Lanague
```
Comment (AdiZav) [inline .github/workflows/offensive_language_actions.yml:1]: Needs refactoring - Lanague -> language

Comment (myakove) [reply .github/workflows/offensive_language_actions.yml:1]: SCK

**Comment by myakove** [comment]
Going to merge this to see if this fix CI
if any change is needed, I will do it on another PR

**Pull request merged**